nucliadb/nucliadb/search/search/chat/prompt.py

```diff
@@ -210,7 +210,11 @@ async def get_resource_field_extracted_text(
     if resource is None:
         return None
 
-    field_type, field_key = field_id.split("/")
+    try:
+        field_type, field_key = field_id.strip("/").split("/")
```

Comment:
We added `.strip("/")` here to remove leading/trailing `/`. Is it really needed? It looks like it makes the validation less strict, which we may want or not.

If this is the case we may want to have a test to ensure leading/trailing `/` are ignored.

Reply:
Yes! I added the strip so that trailing/leading / were ignored. Will add the test!!